08-regression2.Rmd

```diff
@@ -378,9 +406,10 @@ lm_mult_test_results <- lm_fit %>%
 lm_mult_test_results
```

Comment:
We print these results out, but we don't interpret them or say anything about them here...